client/my-sites/patterns/hooks/use-patterns.ts

```diff
@@ -17,6 +27,37 @@ export function getPatternsQueryOptions(
 		},
 		staleTime: 5 * 60 * 1000,
 		...queryOptions,
+		select( patterns: Pattern[] ) {
+			const patternCounts: PatternCounts = {
+				page: {},
+				regular: {},
```

Comment:
```suggestion
				[ CATEGORY_PAGE ]: {},
				[ CATEGORY_REGULAR ]: {},
```
Should we be using those constants there as well?

Reply:
I thought about that as well but was hesitant to follow that route: these are not categories per se but more pattern types. However we could use the `PAGES` and `REGULAR` [types](https://github.com/Automattic/wp-calypso/blob/2cc0abfd45d41b68a63afff7c63949e1e31be8e0/client/my-sites/patterns/types.ts#L18-L21) for that, I'll update that code.